dicom-archive/profileTemplate.pl

```diff
@@ -45,29 +45,24 @@ sub getSubjectIDs {
      #      b. 0 if imaging pipeline should not create the visit label (when visit label has not been created yet in the database. 
     if ($patientName =~ /PHA/i or $patientName =~ /TEST/i) {
 
-        $subjectID{'CandID'}    = NeuroDB::MRI::my_trim(NeuroDB::MRI::getScannerCandID($scannerID, $db));
-        $subjectID{'visitLabel'}= NeuroDB::MRI::my_trim($patientName);
-        $subjectID{'createVisitLabel'} = 1; 
+        $subjectID{'CandID'}     = NeuroDB::MRI::my_trim(NeuroDB::MRI::getScannerCandID($scannerID, $db));
+        $subjectID{'visitLabel'} = NeuroDB::MRI::my_trim($patientName);
+
+        $subjectID{'createVisitLabel'} = 1;
 
      # If patient match PSCID_DCCID_VisitLabel
      # Determine PSCID, DCCID and visitLabel based on patient name
     } elsif ($patientName =~ /([^_]+)_(\d+)_([^_]+)/) {
 
-        $subjectID{'PSCID'}         = NeuroDB::MRI::my_trim($1);
-        $subjectID{'CandID'}        = NeuroDB::MRI::my_trim($2);
-        $subjectID{'visitLabel'}    = NeuroDB::MRI::my_trim($3);
-        $subjectID{'createVisitLabel'} = 0; 
+        $subjectID{'PSCID'}      = NeuroDB::MRI::my_trim($1);
+        $subjectID{'CandID'}     = NeuroDB::MRI::my_trim($2);
+        $subjectID{'visitLabel'} = NeuroDB::MRI::my_trim($3);
 
-        # Note, this function will ensure that the PSCID/CandID information refers to a valid candidate
-        # and it will check that the Visit label exists in the Visit_Windows table
-        if(!NeuroDB::MRI::subjectIDIsValid($subjectID{'CandID'}, $subjectID{'PSCID'}, $subjectID{'visitLabel'}, $dbhr)) {
```

Comment:
removed this as removed the function subjectIDIsValid from `MRI.pm` as it has been refactored to use the functions in `MRIProcessingUtilities.pm` only. There was duplication of validation (one done in `MRI.pm` and the other done in `MRIProcessingUtilities`)